docs/getting-started/overview.rst

```diff
@@ -1,6 +1,113 @@
+access to Windows' native user interface controls. WPF, on the other hand, allows you to exercise 
+much more control over the look and feel of your application. Both of them allow for building very 
+rich desktop applications that run on Windows, and you should pick the one that is suited for your 
+use case. 
+
+`Windows Communication Foundation (WCF) <https://msdn.microsoft.com/en-us/library/ms731082(v=vs.110).aspx>`_ 
+is a set of libraries that comprise the middleware services stack on .NET Framework. 
+It allows you to create services that can communicate through various supported 
+protocols using various data formats, and that can be hoster in any process 
+you choose. This leads to one of the major features of WCF: your services are 
+not tied to any particular hosting strategy or approach.
+
+**ASP.NET** is the .NET web framework. Being a very rich framework, it has several distinct pieces 
+which are used to produce modern and high-performance web applications. 
+`ASP.NET Web Forms <http://www.asp.net/web-forms>`_ is a set of tools geared primarily towards 
+developer productivity, allowing quick turnaround on web applications with a drag-and-drop surface 
+reusing web controls for everything from loging in to data binding. 
+`ASP.NET MVC <http://www.asp.net/mvc>`_ allows for a different approach, one that gives you greater 
+control over the entire pipeline, from HTTP layer to the user interface. 
+`ASP.NET WebAPI <http://www.asp.net/web-api>`_ is a convention-based framework for creating REST 
+services. It allows you to stand up a REST endpoint extremely fast. Finally, 
+`SignalR <http://www.asp.net/signalr>`_ allows you to provide push-based communication to your web 
+applications using `WebSocket <https://en.wikipedia.org/wiki/WebSocket>`_ protocol. 
+
+
+.NET Native
+-----------
+
+.NET Native is not so much an "edition" of the .NET platform as it is a set of tools that 
+allow developers to have different build outputs. The normal .NET source compilation process takes 
+the source code written in one of the .NET languages (such as C#, Visual Basic, F# etc.) and 
+produces something called "Intermediate Language". IL is then picked up by the runtime, 
+and Just-In-Time compiled at run-time to machine code. 
+
+.NET Native is an additional step in this pipeline which takes the IL and compiles it 
+**Ahead-of-Time** (AOT) to machine code, so that when the code is executed, there is 
+only "native" code running. This means that the resulting binary is what the OS executes; 
+there is no JIT-ing, no runtime compilation. This leads to better performance, as well as 
+some security benefits. 
+
+.NET Native is the set of tools used to build .NET **Universal Windows Platform (UWP)** 
+applications. 
+
+Supported operating systems
+---------------------------
+
+.NET Framework is supported only on Windows operating system. .NET Native is, 
```

Comment:
I think that is reading a little bit much into it. The sentence states that .NET Framework works only on Windows to avoid any confusion on which edition works on what OS.